plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php

```diff
@@ -44,7 +44,7 @@ public function initialize(array $config)
     {
         $success = parent::initialize($config);
 
-        if (empty($this->_config['baseUrl']) && strpos($this->getConfig('path'), WWW_ROOT . DS) === 0) {
```

Comment:
The trailing directory separator was there to avoid errors in case there's another directory whose path begins with `BEDITA_ROOT/webroot`. For instance, if i want to store my static files under `BEDITA_ROOT/webroot2` the automatic detection of base URL should fail. However this is quite an edge case, and we may want to ignore it.

Reply:
You're right but `WWW_ROOT` is defined as `ROOT . DS . 'webroot' . DS` so it already contains dir separator the end.

Reply:
Oh, right. Sorry, I hadn't noticed that.